webapp/src/translations/languages/ru.json

```diff
@@ -0,0 +1,575 @@
+      "header": "Центр обработки блоков DFI",
+      "detail": "Похоже, вы случайно здесь наткнулись. Именно здесь наши работники следят за тем, чтобы каждый блок DFI был правильно прикреплен к цепи. Поскольку это закрытая зона, мы просим вас развернуться.",
+      "goBack": "Назад"
+    },
+    "exchangePage": {
+      "title": "Обменник – Блокчейн-Клиент DeFi",
+      "exchange": "Обменник",
+      "section": "Раздел",
+      "footerBar": "Нижняя стойка"
+    },
+    "helpPage": {
+      "title": "Помощь – Блокчейн-Клиент DeFi",
+      "help": "Помощь",
+      "section": "Раздел"
+    },
+    "masterNodes": {
+      "masterNodesPage": {
+        "title": "Мастерноды – Блокчейн-Клиент DeFi",
+        "createMasterNode": "Создать",
+        "masterNodes": "Мастерноды",
+        "statistics": "Статистика",
+        "list": "Список",
+        "weeklyIncome": "Еженедельный доход",
+        "volume": "Объем",
+        "marketCap": "Рыночная капитализация",
+        "returnPerAnnum": "Доходность в год",
+        "paidRewards": "Платные награды",
+        "rewardFrequency": "Частота вознаграждения",
+        "activeMasterNodes": "Активные мастерноды",
+        "supply": "Поставка",
+        "lockedInCollateral": "Заблокировано в залог",
+        "costPerMasterNode": "Стоимость за мастерноду",
+        "masternodeWorth": "Стоимость мастерноды",
+        "searchBar": "Поиск мастернод"
+      },
+      "masterNodesList": {
+        "ownerAddress": "Адрес владельца",
+        "operatorAddress": "Адрес оператора",
+        "registered": "Зарегистрировано",
+        "mintedBlocks": "Отчеканенные блоки",
+        "noMasterNodes": "Нет доступных мастернод",
+        "loading": "Загрузка..."
+      },
+      "createMasterNode": {
+        "title": "Создать мастерноду",
+        "masterNodeName": "Имя мастерноды",
+        "lackOfBalanceMsg": "Не удалось создать мастерноду из-за нехватки средств. Для создания мастерноды требуется минимум 1 миллион DFI",
+        "confirmationText": "Вы уверены, что хотите создать мастерноду?",
+        "restartNodeConfirmationText": "Вы уверены, что хотите перезапустить ноду?",
+        "createNodeButton": "Создать мастерноду",
+        "noButtonText": "Отменить",
+        "yesButtonText": "Продолжить",
+        "masternodeOperator": "Оператор мастерноды",
+        "masternodeOwner": "Владелец мастерноды",
+        "backToMasternodePage": "Вернуться на главную страницу мастерноды",
+        "restartNodeButton": "Перезапустить ноду",
+        "masterNodeSuccess": "Вы можете вручную установить конфигурацию мастерноды в define.conf или вы можете автоматически установить эту конфигурацию мастерноды, перезапустив ноду.",
+        "collaterAddress": "Дополнительный адрес",
+        "masternodeOperatorNotOptional": "Адрес оператора",
+        "masternodeOperatorOptional": "Адрес оператора (необязательно)",
+        "noticeMasternodeOperator": "Если оператор мастерноды отличается от владельца",
```

Comment:
How this phrase sounds on English?

Reply:
If masternode operator is different from owner

Reply:
@DeFiDev what change do you want here, can you explain a bit?